classes/tdx-encrypted.bbclass

```diff
@@ -24,12 +24,6 @@ TDX_ENC_KEY_DIR ?= "/var/local/private/.keys"
 # encryption key blob file name
 TDX_ENC_KEY_FILE ?= "tdx-enc-key.blob"
 
-# Type of encryption
-# This variable defines what will be encrypted
-# Available options:
-#    partition -> encrypt a full partition
-TDX_ENC_STORAGE_TYPE ?= "partition"
```

Comment:
Any particular reason to remove this variable, besides not being used for now? My initial idea was to support other types of encryption, like `file` based encryption. Hopefully I will be able to implement that soon, so if you don't mind, I would like to keep it. :)

Reply:
If we have intentions of using it in the near future I don't see any problem; otherwise I'd add it when we introduce the feature using it. I'll put it back then...

Reply:
Done